recipe/meta.yaml

```diff
@@ -190,12 +191,7 @@ outputs:
       skip: true  # [win or s390x or cross_target_platform not in ("linux-64", "linux-aarch64", "linux-ppc64le")]
     requirements:
       run:
-        - libgcc-devel_{{ cross_target_platform }} {{ gcc_version }}.*
-        # C++ compiler on linux needs a stdlib
-        - libstdcxx-devel_{{ cross_target_platform }}  {{ gcc_version }}.*
```

Comment:
Don't we need libstdcxx for clangxx_impl?

Reply:
`gxx_impl` also [depends](https://github.com/conda-forge/ctng-compilers-feedstock/blob/55c5af125402e8a59e38fdeb72d47dafdc7e67f5/recipe/meta.yaml#L302) on it, so adding that back...